tools/webpack/packages.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * External dependencies
+ */
+const LiveReloadPlugin = require( 'webpack-livereload-plugin' );
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const postcss = require( 'postcss' );
+
+const { join } = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const LibraryExportDefaultPlugin = require( '@wordpress/library-export-default-webpack-plugin' );
+
+const baseDir = join( __dirname, '../../' );
+
+/**
+ * Given a string, returns a new string with dash separators converedd to
+ * camel-case equivalent. This is not as aggressive as `_.camelCase` in
+ * converting to uppercase, where Lodash will convert letters following
+ * numbers.
+ *
+ * @param {string} string Input dash-delimited string.
+ *
+ * @return {string} Camel-cased string.
+ */
+function camelCaseDash( string ) {
+	return string.replace(
+		/-([a-z])/g,
+		( match, letter ) => letter.toUpperCase()
+	);
+}
+
+module.exports = function( env = { environment: 'production' } ) {
+	const mode = env.environment;
+	const suffix = mode === 'production' ? '.min': '';
+
+	const gutenbergPackages = [
+		'a11y',
+		'api-fetch',
+		'autop',
+		'blob',
+		'blocks',
+//		'block-library', // Not on npm yet.
+		'block-serialization-default-parser',
+//		'block-serialization-spec-parser', // No build-module folder.
```

Comment:
It's fine to not have `build-module` in a package. Webpack should default to `build` automatically when you `import "package"`

Reply:
`block-serialization-spec-parser` - as far as I know we use only `block-serialization-default-parser` - the other exists only as an alternative solution which was replaced by `block-serialization-default-parser`.

Reply:
Using `@wordpress/[package]` doesn't work because is is also an external. In Gutenberg this is not a problem because the packages are referred to by their relative path. Given we want to have 1 monorepo for the whole of WordPress in the end I think we can hard-code `build-module` for now.

Reply:
I will fix it for those two packages though.